vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -33,11 +33,19 @@ For more information, see the [Dash documentation](https://dash.plotly.com/exter
 
 ## Add a logo image
 
-Vizro automatically incorporates the dashboard logo in the top-left corner of each page if an image named
-`logo.<extension>` is present within the assets folder.
+### Single logo
 
-Image types of `apng`, `avif`, `gif`, `jpeg`, `jpg`, `png`, `svg`, and `webp` are supported.
+Vizro automatically incorporates a logo in the top-left corner if an image named `logo.<extension>` is present in the
```

Comment:
```suggestion
Vizro automatically incorporates a logo in the top-left corner of an image named `logo.<extension>` is present in the
```

Reply:
Ah, I see that this might have been confusing. I think above is technically not correct though, but would you agree with the following:

Vizro automatically incorporates a logo in the top-left corner **of a dashboard** **if** an image named `logo.<extension>` is present in the ..

Reply:
Ah, I see. I thought it was unlike you to submit a PR with a typo 🙇 

Let's invert the sentence.